assets/rpvoicechat/lang/ru.json

```diff
@@ -19,20 +19,38 @@
     "Command.Talk.Success": "Расстояние разговора установлено на {0}",
     "Command.Whisper.Desc": "Устанавливает расстояние шёпота в блоках",
     "Command.Whisper.Success": "Расстояние шёпота установлено на {0}",
-    "Command.Info.Desc": "Отображает текущие расстояния для крика, разговора и шёпота",
-    "Command.Info.Success": "Расстояние шёпота: {0} блоков\nРасстояние разговора: {1} блоков\nРасстояние крика: {2} блоков",
+    "Command.Info.Desc": "Отображает текущие расстояния для крика, разговора и шёпота и другие значения",
+    "Command.Info.Success": "Расстояние шёпота: {0} блоков\nРасстояние разговора: {1} блоков\nРасстояние крика: {2} блоков\nОтображать имена игроков: {3}\nКодировать звук: {4}",
     "Command.Reset.Desc": "Сбрасывает расстояния для крика, разговора и шёпота к значениям по умолчанию",
     "Command.Reset.Success": "Расстояния для крика, разговора и шёпота сброшены к значениям по умолчанию",
+    "Command.ForceNameTags.Desc": "Включает/выключает принудительное отображение имён игроков при разговоре",
+    "Command.ForceNameTags.Help": "Если имена игроков включены всё время, эта настройка ни на что не влияет. Значение по-умолчанию: <strong>true</strong>",
+    "Command.ForceNameTags.Success.Enabled": "Имена игроков БУДУТ отображаться при разговоре, даже если они были скрыты",
+    "Command.ForceNameTags.Success.Disabled": "Имена игроков НЕ БУДУТ отображаться при разговоре, если они были скрыты",
+    "Command.EncodeAudio.Desc": "Включает/выключает кодирование звука. Дополнительную информацию смотри в /help",
```

Comment:
Just `/help` isn't sufficient to get info about specific command, it should either be `/help rpvc encodeaudio` or just generally mention to use help command without explicitly telling the command

Reply:
```suggestion
    "Command.EncodeAudio.Desc": "Включает/выключает кодирование звука. Дополнительную информацию читай в <a href=\"chattype:///help rpvoicechat encodeaudio\">справке</a>.",
```